net-dialup/accel-ppp/files/accel-ppp-1.12.0-ucontext.patch

```diff
@@ -0,0 +1,17 @@
+# Use ucontext, this is specific to musl libc
+#
+# Alpine patch:
+# https://git.alpinelinux.org/aports/tree/community/accel-ppp/0001-ucontext.patch
+#
+# Closes: https://bugs.gentoo.org/828906
+--- a/accel-pppd/CMakeLists.txt
++++ b/accel-pppd/CMakeLists.txt
+@@ -113,7 +113,7 @@ ADD_EXECUTABLE(accel-pppd
+ 	main.c
+ )
+
+-TARGET_LINK_LIBRARIES(accel-pppd triton rt pthread ${crypto_lib} pcre)
++TARGET_LINK_LIBRARIES(accel-pppd triton rt pthread ${crypto_lib} pcre ucontext)
```

Comment:
You sure this works on glibc?

Reply:
Hhh

Reply:
If it builds, it should be OK.

Reply:
The ucontext inclusion was conditional and I made the mistake of testing on Glibc while libucontext installed as a result couldn't catch that bug on glibc. Should be fixed with latest push. libucontext inclusion is now conditional.